docs/version0.44.md

```diff
@@ -31,6 +31,7 @@ The following new features and notable changes since version 0.43.0 are included
 - [XL C++ Runtime 16.1.0.7 or later required for AIX OpenJ9 builds on OpenJDK 8](#xl-c-runtime-16107-or-later-required-for-aix-openj9-builds-on-openjdk-8)
 - [New `-XX:[+|-]ShowUnmountedThreadStacks` option added](#new-xx-showunmountedthreadstacks-option-added)
 - [VMID query in the `jcmd` tool enhanced](#vmid-query-in-the-jcmd-tool-enhanced)
+- [Field names in `J9BuildFlags` changed](#field-names-in-j9buildflags-changed)
```

Comment:
I think this should mention DDR so that someone with a DDR extension will know to read about the change.